### PR TITLE
Who's using GoChain

### PR DESCRIPTION
**Pull request by treeder**

### CODE REVIEW

**Comment by treeder** [comment]
Let me know if I'm missing someone or if there's any corrections.

File: whos-using-gochain.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* [MyCrypto](https://mycrypto.com)
+* [Blockfolio](https://blockfolio.com/)
+* [Trust Wallet](https://trustwalletapp.com/)
+* [TurboWallet](https://turbowallet.io)
+
+## Exchanges
+
+* [Bittrex](https://bittrex.com)
+* [Binance](https://binance.com)
+* [KuCoin](https://kucoin.com)
```
Comment (jmank88) [inline whos-using-gochain.md]: Upbit?

File: whos-using-gochain.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+All of these great companies are built on or are building on GoChain!
+
+* [Veritoken](https://veritoken.io/)
+* [Solaster](https://solaster.io)
+* [Etherprise](https://etherprise.io)
+* [Job.com](https://job.com)
+* [PrimaBlock](https://primablock.com)
+* [MultiSender](https://multisender.app/)
+* [Monarch](https://monarchtoken.io/)
```
Comment (jmank88) [inline whos-using-gochain.md]: TrueDeck?

**Comment by ialexander28** [comment]
Looks good to me!  Btw, why did this come to my personal email?  Haha

**Pull request merged**